src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -30,8 +33,11 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       return;
     }
 
-    String aboutMe = String.format("This is %s's about me.", user);
-    response.getOutputStream().println(aboutMe);
+    User userData = datastore.getUser(user);
+
+    if (userData == null || userData.getAboutMe() == null) {
+      return;
+    }
```

Comment:
This function looks incomplete. It seems like you are never doing anything with the response data and only return null.